seta-react/src/pages/CommunitiesPage/pages/contexts/notifications-context.tsx

```diff
@@ -58,52 +46,6 @@ export const NotificationsProvider = ({ children }: ChildrenProp) => {
     }
   }, [isMounted, notifications])
 
-  useEffect(() => {
-    getPermissions().then(response => {
-      setPermissions(response)
-    })
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
-
-  const BASE_URL = environment.baseUrl
-  const apiConfig: AxiosRequestConfig = {
-    baseURL: BASE_URL
-  }
-
-  const getPermissions = async (): Promise<UserPermissions> => {
-    const { data } = await api.get<UserPermissions>('/me/permissions', apiConfig)
-
-    setPermissions(data)
-
-    return data
-  }
-
-  const getMembershipRequests = async (): Promise<MembershipRequest[]> => {
-    const list: MembershipRequest[] = []
-
-    permissions?.community_scopes
-      ?.filter(
-        scope =>
-          scope.scopes.includes('/seta/community/manager') ||
-          scope.scopes.includes('/seta/community/owner')
-      )
-      .forEach(async item => {
-        await community_api
-          .get<MembershipRequest[]>(
-            `${environment.COMMUNITIES_API_PATH}/${item.community_id}/requests`
-          )
-          .then(response => {
-            list.push(...response.data)
-
-            return response.data
-          })
-      })
-
-    setMemberships(list)
-
-    return list
-  }
-
   const getNotificationRequests = async () => {
     let count = 0
     const result = await community_api.get<NotificationsResponse[]>(`/notifications/`)
```

Comment:
Is there a reason why the api calls don't run through the `api` object in `~/api/api.ts` which is using a custom configuration for axios with interceptors for authentication and logging?